.pre-commit-config.yaml

```diff
@@ -14,6 +14,6 @@ repos:
     hooks:
       # Run the linter
       - id: ruff
-        args: [--fix, --exit-non-zero-on-fix, --select, I]
+        args: [--fix, --exit-non-zero-on-fix, --extend-select, I]
```

Comment:
If you add a `ruff.toml` file like the one on [udata](https://github.com/opendatateam/udata/pull/3118/files#diff-26f436cb29eecc7bb4f9066747b43df8da81ac808955ce95c109f5d7aa778989), you shouldn't need to specify the options like that.